configs/eslint-config-compass/index.js

```diff
@@ -57,6 +57,13 @@ const testTsOverrides = {
   },
 };
 
+const testHelpersOverrides = {
+  files: ['**/test/**/*'],
```

Comment:
Would it make sense to add `**/test/**/*.js` and `**/test/**/*.ts` patterns to the existing `testJsOverrides` and `testTsOverrides` file patterns, respectively?

Reply:
Yea - I initially did that but it started creating other eslint problems in multiple packages. From the initial analysis it looked like the pattern that I added was conflicting somehow with other patterns. Some errors were quite cryptic to debug and I wanted a quick turn around so I decided to keep it separate here. I can try to debug those issues further if we should definitely do the other one.

Reply:
Hm, it works fine for me (diff I tried: https://gist.github.com/addaleax/86a632e15db6805b787bf99e95f3911c)

Do you remember the actual error message here? This probably isn't something you need to spend time on for this PR specifically, no, but I'm happy to take a deeper look if I find some time to do so

Reply:
Ok so the issue was that I was using `**/test/**/*` which was most likely also including non-js/ts files and for that reason the eslint parser was failing on those files with this cryptic error. With your pattern it all works fine. Thank you 🙌 I just added this commit.